azerothcore/mod-congrats-on-level
# feat. Move awards to database and enable races/classes

## Changes Proposed:
- By moving the awards to the database, more can be added, and by the way, the attributes of class and race were added, in case you want to give specific awards for those values. If it is 0, it is given to all, and then, depending on the value chosen, it is given to their respective owners.
- By disabling the CongratsPerLevelEnable, the message is no longer sent in the chat, each time the player changes level. Requires server reload. Does not work with the `.reload config` command.

## Issues Addressed:

- Closes https://github.com/azerothcore/mod-congrats-on-level/issues/25
- Closes https://github.com/azerothcore/mod-congrats-on-level/issues/22

## SOURCE:


## Tests Performed:

- Windows 10
- In-game

## How to Test the Changes:


1. The default awards, set in the database, are those previously delivered by the module. They can be changed or even added. Note that the table has 3 primary attributes, so the awards cannot be repeated infinitely. If necessary, this condition can be removed.
2. Simply start leveling up and you will receive rewards, gold, some items, and some spells.
